Doc/library/ssl.rst

```diff
@@ -2158,9 +2158,9 @@ Visual inspection shows that the certificate does identify the desired service
                 (('commonName', 'www.python.org'),)),
     'subjectAltName': (('DNS', 'www.python.org'),
                        ('DNS', 'python.org'),
-                       ('DNS', 'pypi.python.org'),
+                       ('DNS', 'pypi.org'),
                        ('DNS', 'docs.python.org'),
-                       ('DNS', 'testpypi.python.org'),
+                       ('DNS', 'testpypi.org'),
```

Comment:
should this be test.pypi.org ? testpypi.org gives me a 404

Reply:
I can't fix it in this PR, because I don't have the access on the cpython repository.

Reply:
I don't understand. How can you be able to make this change but not able to modify what change you are making?

Reply:
Because this PR has been created by @miss-islington and I am not the author of this PR.

Reply:
I don't think it's much of a problem that `testpypi.org` doesn't exist. This is just an example, it's outdated in other ways as well, and I don't think anyone would expect us to keep it up to date.
If you disagree, please bring it up in https://github.com/python/cpython/pull/6814 or in a new issue. The `testpypi.org` example is already in master.